inc/class-shortcode-ui.php

```diff
@@ -137,10 +155,12 @@ public function action_wp_enqueue_editor() {
 	 *
 	 * @return null
 	 */
-	public function action_print_media_templates() {
+	public function print_templates() {
```

Comment:
D'oh - making this consistent with the action name it's hooked to rather than just reverting the name.